TestAssets/TestProjects/PortableApp/project.json

```diff
@@ -0,0 +1,18 @@
+{
+ 
+  "buildOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {},
+  "frameworks": {
+    "netcoreapp1.0": {
+      "dependencies": {
+        "Microsoft.NETCore.App": {
+          "type": "platform",
+          "version": "1.0.0-rc3-*"
+        },
+        "Newtonsoft.Json": "9.0.1-beta1"
```

Comment:
Maybe the app should actually load Newtonsoft up and use it then.  To ensure the assembly can be loaded and executed at runtime. It would make it more explicit that this dependency is needed, and not a mistake.

If I was in here refactoring stuff, I would delete this dependency because the app doesn't use it.

---
In reply to: [66351095](https://github.com/dotnet/core-setup/pull/97#discussion_r66351095) [](ancestors = 66351095,66346538)